packages/plugin-core/src/utils/ExtensionUtils.ts

```diff
@@ -439,6 +442,70 @@ export class ExtensionUtils {
     if (previewTheme !== undefined) {
       _.set(trackProps, "previewTheme", previewTheme);
     }
+
+    try {
+      const allExtensions = vscode.extensions.all;
+      const extensionsDetail = allExtensions
+        .filter((extension) => {
+          return !extension.packageJSON.isBuiltin;
+        })
+        .map((extension) => {
+          const { packageJSON } = extension;
+          const { id, version } = packageJSON;
+          return { id, version };
+        });
+      if (extensionsDetail && extensionsDetail.length > 0) {
+        _.set(trackProps, "extensionsDetail", JSON.stringify(extensionsDetail));
+        _.set(trackProps, "numExtensions", extensionsDetail.length);
+      }
+    } catch (error) {
+      // something went wrong don't track extension detail
+    }
+
+    try {
+      const VSCODE_INSTALLATIONS = [
```

Comment:
can we get this from the current vscode installation? 

`codeFoldersCreated` should be a single number that shows when the current initialized version of vscode was created

Reply:
I've changed it so that we infer the location more intelligently. It will have more coverage (mostly likely will cover portable installs as well) and doesn't rely on trying and failing on known default locations.